var/spack/repos/builtin/packages/dakota/package.py

```diff
@@ -44,6 +44,7 @@ class Dakota(CMakePackage):
     depends_on('mpi', when='+mpi')
 
     depends_on('python')
+    depends_on('perl-data-dumper')
```

Comment:
Is this a dependency for all versions or only those from a specific point on?

Also, if this is only a build dependency you could add `, type='build'`.

Reply:
Updated for version 6.12 and above and added your suggested type='build'